css/index.css

```diff
@@ -1 +1,396 @@
+	align-items: center;
+	width: 100%;
+
+}
+
+.cta .cta-text {
+  display: flex;
+	flex-direction: column;
+  font-size: 7.7rem;
+  text-align: center;
+  align-items: center;
+	padding-top: 3rem;
+  align-self: baseline;
+	width: 50%;
+}
+
+.cta .cta-text button {
+  display: flex;
+  justify-content: center;
+  background: white;
+  margin: 1rem;
+	padding: 0.7rem;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  cursor: pointer;
+  width: 40%;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: flex;
+
+}
+
+.main-content {
+	display: flex;
+  border-top: 2px solid black;
+  margin-top: 6.6rem;
+  padding-top: 3.3rem;
+	flex-direction: column;
+	width: 100%;
+}
+.main-content h4 {
+	font-size: 1.6rem;
+}
+.main-content p {
+	padding-top: .6rem;
+	font-size: 1.6rem;
+}
+
+.main-content .top-content {
+  display: flex;
+
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  display: flex;
+  margin-top: 3.5rem;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  margin-top: 3.7rem;
+}
+
+
+
+.contact {
+  margin-top: 4rem;
+  border-top: 2px solid black;
+	padding-top: 3rem;
+	font-size: 1.6rem;
+}
+
+.contact p {
+  margin: 2rem 0;
+}
+
+footer {
+  margin: 3.2rem 0;
+  text-align: center;
+  font-size: 1.4rem;
+}
+
+/* Tablet view */
+@media (max-width: 800px) {
+  .container {
+		max-width: 880px;
+		width: 100%;
+	}
+	header {
+    display: flex;
+    flex-direction: column-reverse;
+		justify-content: center;
+		align-content: center;
+		align-items: center;
+		margin-bottom: 3rem;
+		width: 100%;
+	}
+	header .logo {
+		display: flex;
+    width: 25%;
+		justify-content: center;
+		margin-top: 3rem;
+		margin-bottom: 3rem;
+		animation-duration: 0.7s;
+		animation-name: slidein;
+	}
+
+	@keyframes slidein {
+		from {
+			margin-left: 100%;
+			width: 70%;
+		}
+		to {
+			margin-right: 0%;
+			width: 30%;
+		}
+	}
+
+	header nav {
+		align-items: flex-end;
+		width: 100%;
+		justify-content: space-around;
+	}
+	.cta {
+   display: flex;
+	 justify-content: center;
+	 margin-top: 2rem;
+	 margin-bottom: 2rem;
+	}
+	 .cta img {
+		 display: none;
+	 }
+	 .cta p {
+		 padding: 0.5rem;
+	 }
+	 .cta-text {
+		 padding-bottom: 3rem;
+	 }
+	 .main-content {
+		 display: flex;
+		 justify-content: space-between;
+		 width: 100%;
+		 text-size: 2em;
+	 }
+	 .contact {
+		 margin: 2rem 0;
+	 }
+	 .middle-img {
+		 display: flex;
+		 align-content: center;
+		 justify-content: center;
+		 width: 100%;
+		 }
+	}
+
+
+/* Mobile view */
+@media (max-width: 500px) {
+	.container {
+		max-width: 500px;
+			width: 100%;
+			margin: 0 auto;
+	}
+	header .logo {
+		width: 40%;
+		}
+  header nav {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: center;
+		align-content: center;
+		width: 100%;
+	}
+	header nav a {
+		display: flex;
+		font-size: 2em;
+		border: 1px solid black;
```

Comment:
To look more like the design file, I would use border-bottom here. Then you can either add a border bottom to the logo or do something like use a pseudo-class `header nav:first-child { ... }`.